src/mame/sinclair/glukrs.cpp

```diff
@@ -14,18 +14,9 @@
 #include "glukrs.h"
 
 glukrs_device::glukrs_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock)
-		: mc146818_device(mconfig, GLUKRS, tag, owner, clock)
+		: mc146818_device(mconfig, GLUKRS, tag, owner, 32'768)
```

Comment:
Ignoring the clock frequency is bad form.  Wouldn’t this use a 32768 Hz watch crystal for timing in practice?  It should still be passed in from machine configuration.

Reply:
I presume the OSC is internal here? Then the driver mcfg should not 'know' the clock and should not pass any value to it.

Reply:
I doubt it's internal, but every clock chip I've read the datasheet for wants a 32768 Hz crystal and several of them say any other value is not guaranteed to work.

Reply:
This device is specific to ZX Evolution usage and have clock "defined". I would like to avoid ability to modify this clock value as it implies additional configuration in RTC registers to work correctly.
As a option I can add validation to device which will check if supplied clock is desired.